src/style/variables.css

```diff
@@ -0,0 +1,22 @@
+@import url(https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap);
```

Comment:
```suggestion
@import url(https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap);
```

Reply:
import also 400 and 500 font-weight with this link, so it can. be applied to the text

<img width="252" alt="Screenshot 2022-06-13 at 16 13 15" src="https://user-images.githubusercontent.com/57598825/173361637-f05a856d-0b9a-47e2-a91c-811d0e785422.png">
<img width="300" alt="Screenshot 2022-06-13 at 16 13 24" src="https://user-images.githubusercontent.com/57598825/173361665-75e2d6f1-7af9-4945-bbc0-eecc8f5dff69.png">